zkevm-circuits/src/keccak_circuit/keccak_padding_byte_2col_lookup.rs

```diff
@@ -52,6 +52,15 @@ impl<F: Field> PaddingCombinationsConfig<F> {
         }
     }
 
+    pub(crate) fn assign_table_row(&self, mut table: Table<'_, F>, row_id: usize, 
```

Comment:
Change to `table: &mut Table<'_, F>`, you're trying to change the table object that's passed into the function. So you want to pass in a mutable reference.

Reply:
Because the function is only used in a single function here it's actually a bit cleaner to use a closure like this (included after `|mut table: Table<'_, F>| {`):
```
let mut assign_table_row  = |
                  row_id: usize,
                     byte_val: u64,
                     is_pad_val: u64,
                     is_pad_start_val: u64,
                     is_pad_end_val: u64,
                    | -> Result<(), Error> {
  table.assign_cell(|| format!("byte_col_[row={}]", row_id), self.byte_col, row_id, || Ok(F::from(byte_val)))?;
  table.assign_cell(|| format!("is_padding_col_[row={}]", row_id), self.is_padding_col, row_id, || Ok(F::from(is_pad_val)))?;
  table.assign_cell(|| format!("is_padding_start_col_[row={}]", row_id), self.is_padding_start_col, row_id, || Ok(F::from(is_pad_start_val)))?;
  table.assign_cell(|| format!("is_padding_end_col_[row={}]", row_id), self.is_padding_end_col, row_id, || Ok(F::from(is_pad_end_val)))?;

  Ok(())
};
```

Which you can then use like:
```
assign_table_row(i as usize, i, 0, 0, 0)?;
```

The benefit is that this can automatically capture data in the outside function so no need to pass it around.